catalyst_voices/Earthfile

```diff
@@ -52,6 +52,12 @@ check-package-publishing:
     FROM +builder
     DO flutter-ci+PUBLISH_DRY_RUN
 
+# Runs the license checks for Catalyst Voices.
+check-license:
+    FROM +builder
+    RUN dart pub global activate license_checker
```

Comment:
It feels like this line should be moved to catalyst-ci, below:

<img width="948" alt="Screenshot 2024-09-23 at 12 50 21" src="https://github.com/user-attachments/assets/51ee4344-dad8-4af3-89fe-38c817c7dd3b">

Reply:
Yeah, makes sense.